### PR TITLE
feat: maven 3.8.3

### LINKED ISSUE #230: Add Apache Maven 3.8.3
Hi it would be great to have Maven 3.8.3 as a image as well...

### PR DESCRIPTION
**Pull request by carlossg**
fixes #230

### CODE REVIEW

File: adoptopenjdk-8-openj9-windowsservercore/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM adoptopenjdk:8-openj9-windowsservercore-1809
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ARG USER_HOME_DIR="C:/Users/ContainerUser"
-ARG MAVEN_VERSION=3.8.2
+ARG MAVEN_VERSION=3.8.3
 ARG SHA=59ad2cbd6b7abde34ebedda94ce5631256373718e71b55202035bd1190d0144f071433f78b99e16f1204413b3eb888659e5039009e1ad0106f16332e3c62bced
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
```
Comment (khmarbaise) [inline adoptopenjdk-8-openj9-windowsservercore/Dockerfile:9]: Missing change in SHA

File: amazoncorretto-8-windowsservercore/Dockerfile [hunk 2]
```diff
@@ -15,7 +15,7 @@ RUN Invoke-WebRequest -Uri $('{0}/{1}' -f $env:uri,$env:zip) -OutFile C:/$env:zi
 ENV JAVA_HOME=C:/ProgramData/jdk1.8.0_302
 
 ARG USER_HOME_DIR="C:/Users/ContainerUser"
-ARG MAVEN_VERSION=3.8.2
+ARG MAVEN_VERSION=3.8.3
 ARG SHA=59ad2cbd6b7abde34ebedda94ce5631256373718e71b55202035bd1190d0144f071433f78b99e16f1204413b3eb888659e5039009e1ad0106f16332e3c62bced
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
```
Comment (khmarbaise) [inline amazoncorretto-8-windowsservercore/Dockerfile:20]: Missing change in SHA

File: openjdk-11-windowsservercore/Dockerfile [hunk 3]
```diff
@@ -4,7 +4,7 @@ FROM openjdk:11-jdk-windowsservercore-1809
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ARG USER_HOME_DIR="C:/Users/ContainerUser"
-ARG MAVEN_VERSION=3.8.2
+ARG MAVEN_VERSION=3.8.3
 ARG SHA=59ad2cbd6b7abde34ebedda94ce5631256373718e71b55202035bd1190d0144f071433f78b99e16f1204413b3eb888659e5039009e1ad0106f16332e3c62bced
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
```
Comment (khmarbaise) [inline openjdk-11-windowsservercore/Dockerfile:9]: Missing change in SHA

**Comment by carlossg** [comment]
https://github.com/docker-library/official-images/pull/11044

**Pull request merged**